src/Drifty_CLI.java

```diff
@@ -1,3 +1,4 @@
+import java.awt.*;
```

Comment:
@poshi1865 Is `awt` package required in this program?

Reply:
Umm I don't think so, I did not add any awt classes, let me check once.

Reply:
@poshi1865 Please respond.

Reply:
I resolved it @SaptarshiSarkar12 , it was unused.

Reply:
ok @poshi1865